### ISSUE TITLE
Allow custom annotations for promethues metrics

### ISSUE THREAD

**Issue by rajugupta15** [nats-io/nats-operator#124]
I am using `prometheus.app` annotations to scrape metrics. Can we have option for custom annotations?

https://github.com/nats-io/nats-operator/blob/d6d163e400ba62764f72d8dbc0e84cb7566f15e3/pkg/util/kubernetes/kubernetes.go#L668

**Comment by annismckenzie** [comment]
Shouldn't #111 allow you to do that? It's been merged already.

**Comment by rajugupta15** [comment]
No, Still i am getting `prometheus.io`. Here is my yaml.
```---
apiVersion: "nats.io/v1alpha2"
kind: "NatsCluster"
metadata:
  name: "nats-cluster-metrics"
  namespace: nats-io
spec:
  size: 3
  version: "1.4.1"
  serverImage: "nats"
  pod:
    enableMetrics: true
    metricsImage: "synadia/prometheus-nats-exporter"
    metricsImageTag: "0.1.0"
    annotations:
      prometheus.app/scrape: "true"
      prometheus.app/port: "7777"```

**Comment by rajugupta15** [comment]
Any update on this?

**Comment by wallyqs** [comment]
Yes this should be supported in master but need to push a new image with the changes (which should be soon...)

**Comment by wallyqs** [comment]
Fixed via #111
